blackbox/docs/sql/reference/create_ingest_rule.txt

```diff
@@ -0,0 +1,86 @@
+.. _create-ingest-rule:
+
+======================
+``CREATE INGEST RULE``
+======================
+
+Defines a new ingestion rule.
+
+Synopsis
+========
+.. highlight:: psql
+
+::
+
+    CREATE INGEST RULE rule_name
+    ON source_ident
+    [WHERE condition]
+    INTO table_ident
+
+
+Description
+===========
+
+``CREATE INGEST RULE`` creates a new ingestion rule, applying on a
+``source_ident`` and ingesting into the specified target ``table_ident``.
+
+The ingestion rules will be used by ingestion plugins to filter the data
+streams of its ``source_ident``, the filtered data will then be
```

Comment:
I think this will be clearer if we say
"... ingestion plugins to filter the ``source_ident`` stream. The filtered data will then ... "